### PR TITLE
libsettings with LSN as library [ESD-1116]

### PR DESCRIPTION
**Pull request by pmiettinen**
Test and integrate https://github.com/swift-nav/libsettings/pull/27

UPDATE: update straight to libsettings-v0.1.11

### CODE REVIEW

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v2.1.0-develop-2019030522-7-g274675fc

These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.

### job status
+ [live-roof-650-townsend runs](https://gnss-analysis.swiftnav.com/jobs/id=937643)
+ [live-roof-650-townsend-dropouts-zero-baseline runs](https://gnss-analysis.swiftnav.com/jobs/id=937644)

### gnss-analysis results
At least one run must complete for these links to have data.

#### passfail
+ [live-roof-650-townsend](https://gnss-analysis.swiftnav.com/metrics_preset=pass_fail&scenario=live-roof-650-townsend&summary_type=q50&groupby_key=firmware&firmware_versions=v2.1.0-develop-2019030522-7-g274675fc&build_type=buildroot_pr&display_type=table)
+ [live-roof-650-townsend-dropouts-zero-baseline](https://gnss-analysis.swiftnav.com/metrics_preset=pass_fail&scenario=live-roof-650-townsend-dropouts-zero-baseline&summary_type=q50&groupby_key=firmware&firmware_versions=v2.1.0-develop-2019030522-7-g274675fc&build_type=buildroot_pr&display_type=table)

#### detailed
+ [live-roof-650-townsend](https://gnss-analysis.swiftnav.com/metrics_preset=detailed&scenario=live-roof-650-townsend&summary_type=q50&groupby_key=firmware&firmware_versions=v2.1.0-develop-2019030522-7-g274675fc&build_type=buildroot_pr&display_type=table)
+ [live-roof-650-townsend-dropouts-zero-baseline](https://gnss-analysis.swiftnav.com/metrics_preset=detailed&scenario=live-roof-650-townsend-dropouts-zero-baseline&summary_type=q50&groupby_key=firmware&firmware_versions=v2.1.0-develop-2019030522-7-g274675fc&build_type=buildroot_pr&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v2.1.0-develop-2019030522-7-g274675fc
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v2.1.0-develop-2019030522-7-g274675fc)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v2.1.0-develop-2019030522-7-g274675fc/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v2.1.0-develop-2019030522-7-g274675fc

File: package/libsettings/libsettings.mk [hunk 1]
```diff
@@ -4,11 +4,11 @@
 #
 ################################################################################
 
-LIBSETTINGS_VERSION = v0.1.8
+LIBSETTINGS_VERSION = pmiettinen/esd-1204-parallel-requests
```
Comment (pmiettinen) [inline package/libsettings/libsettings.mk]: Tag libsettings v0.1.11

File: package/libswiftnav/libswiftnav.mk [hunk 2]
```diff
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-LIBSWIFTNAV_VERSION = v0.1.5
+LIBSWIFTNAV_VERSION = master
```
Comment (pmiettinen) [inline package/libswiftnav/libswiftnav.mk]: Tag libswiftnav v0.1.6

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v2.2.0-develop-2019033116-2-g5ba5e251

These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.

### job status
+ [live-roof-650-townsend runs](https://gnss-analysis.swiftnav.com/jobs/id=978706)
+ [live-roof-650-townsend-dropouts-zero-baseline runs](https://gnss-analysis.swiftnav.com/jobs/id=978707)

### gnss-analysis results
At least one run must complete for these links to have data.

#### passfail
+ [live-roof-650-townsend](https://gnss-analysis.swiftnav.com/metrics_preset=pass_fail&scenario=live-roof-650-townsend&summary_type=q50&groupby_key=firmware&firmware_versions=v2.2.0-develop-2019033116-2-g5ba5e251&build_type=buildroot_pr&display_type=table)
+ [live-roof-650-townsend-dropouts-zero-baseline](https://gnss-analysis.swiftnav.com/metrics_preset=pass_fail&scenario=live-roof-650-townsend-dropouts-zero-baseline&summary_type=q50&groupby_key=firmware&firmware_versions=v2.2.0-develop-2019033116-2-g5ba5e251&build_type=buildroot_pr&display_type=table)

#### detailed
+ [live-roof-650-townsend](https://gnss-analysis.swiftnav.com/metrics_preset=detailed&scenario=live-roof-650-townsend&summary_type=q50&groupby_key=firmware&firmware_versions=v2.2.0-develop-2019033116-2-g5ba5e251&build_type=buildroot_pr&display_type=table)
+ [live-roof-650-townsend-dropouts-zero-baseline](https://gnss-analysis.swiftnav.com/metrics_preset=detailed&scenario=live-roof-650-townsend-dropouts-zero-baseline&summary_type=q50&groupby_key=firmware&firmware_versions=v2.2.0-develop-2019033116-2-g5ba5e251&build_type=buildroot_pr&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v2.2.0-develop-2019033116-2-g5ba5e251
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v2.2.0-develop-2019033116-2-g5ba5e251)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v2.2.0-develop-2019033116-2-g5ba5e251/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v2.2.0-develop-2019033116-2-g5ba5e251

**Comment by pmiettinen** [comment]
jenkins build failing due to cached `libswiftnav-master.tar.gz`.

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v2.2.0-develop-2019033116-3-g9b6773b0
There was an error using the HITL API to kick off smoke tests for this commit.
See https://travis-ci.org/swift-nav/piksi_buildroot/builds/514520886

**Comment by anonymous-travis-swiftnav** [comment]
## v2.2.0-develop-2019033116-3-g9b6773b0
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v2.2.0-develop-2019033116-3-g9b6773b0)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v2.2.0-develop-2019033116-3-g9b6773b0/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v2.2.0-develop-2019033116-3-g9b6773b0

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v2.2.0-develop-2019033116-4-g15fda4ba
There was an error using the HITL API to kick off smoke tests for this commit.
See https://travis-ci.org/swift-nav/piksi_buildroot/builds/515064005

**Comment by anonymous-travis-swiftnav** [comment]
## v2.2.0-develop-2019033116-4-g15fda4ba
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v2.2.0-develop-2019033116-4-g15fda4ba)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v2.2.0-develop-2019033116-4-g15fda4ba/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v2.2.0-develop-2019033116-4-g15fda4ba

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v2.2.0-develop-2019033116-7-g8604df4a

These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.

### job status
+ [live-roof-650-townsend runs](https://gnss-analysis.swiftnav.com/jobs/id=982858)
+ [live-roof-650-townsend-dropouts-zero-baseline runs](https://gnss-analysis.swiftnav.com/jobs/id=982859)

### gnss-analysis results
At least one run must complete for these links to have data.

#### passfail
+ [live-roof-650-townsend](https://gnss-analysis.swiftnav.com/metrics_preset=pass_fail&scenario=live-roof-650-townsend&summary_type=q50&groupby_key=firmware&firmware_versions=v2.2.0-develop-2019033116-7-g8604df4a&build_type=buildroot_pr&display_type=table)
+ [live-roof-650-townsend-dropouts-zero-baseline](https://gnss-analysis.swiftnav.com/metrics_preset=pass_fail&scenario=live-roof-650-townsend-dropouts-zero-baseline&summary_type=q50&groupby_key=firmware&firmware_versions=v2.2.0-develop-2019033116-7-g8604df4a&build_type=buildroot_pr&display_type=table)

#### detailed
+ [live-roof-650-townsend](https://gnss-analysis.swiftnav.com/metrics_preset=detailed&scenario=live-roof-650-townsend&summary_type=q50&groupby_key=firmware&firmware_versions=v2.2.0-develop-2019033116-7-g8604df4a&build_type=buildroot_pr&display_type=table)
+ [live-roof-650-townsend-dropouts-zero-baseline](https://gnss-analysis.swiftnav.com/metrics_preset=detailed&scenario=live-roof-650-townsend-dropouts-zero-baseline&summary_type=q50&groupby_key=firmware&firmware_versions=v2.2.0-develop-2019033116-7-g8604df4a&build_type=buildroot_pr&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v2.2.0-develop-2019033116-7-g8604df4a
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v2.2.0-develop-2019033116-7-g8604df4a)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v2.2.0-develop-2019033116-7-g8604df4a/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v2.2.0-develop-2019033116-7-g8604df4a

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v2.2.0-develop-2019033116-7-g8604df4a

These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.

### job status
+ [live-roof-650-townsend runs](https://gnss-analysis.swiftnav.com/jobs/id=983061)
+ [live-roof-650-townsend-dropouts-zero-baseline runs](https://gnss-analysis.swiftnav.com/jobs/id=983062)

### gnss-analysis results
At least one run must complete for these links to have data.

#### passfail
+ [live-roof-650-townsend](https://gnss-analysis.swiftnav.com/metrics_preset=pass_fail&scenario=live-roof-650-townsend&summary_type=q50&groupby_key=firmware&firmware_versions=v2.2.0-develop-2019033116-7-g8604df4a&build_type=buildroot_pr&display_type=table)
+ [live-roof-650-townsend-dropouts-zero-baseline](https://gnss-analysis.swiftnav.com/metrics_preset=pass_fail&scenario=live-roof-650-townsend-dropouts-zero-baseline&summary_type=q50&groupby_key=firmware&firmware_versions=v2.2.0-develop-2019033116-7-g8604df4a&build_type=buildroot_pr&display_type=table)

#### detailed
+ [live-roof-650-townsend](https://gnss-analysis.swiftnav.com/metrics_preset=detailed&scenario=live-roof-650-townsend&summary_type=q50&groupby_key=firmware&firmware_versions=v2.2.0-develop-2019033116-7-g8604df4a&build_type=buildroot_pr&display_type=table)
+ [live-roof-650-townsend-dropouts-zero-baseline](https://gnss-analysis.swiftnav.com/metrics_preset=detailed&scenario=live-roof-650-townsend-dropouts-zero-baseline&summary_type=q50&groupby_key=firmware&firmware_versions=v2.2.0-develop-2019033116-7-g8604df4a&build_type=buildroot_pr&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v2.2.0-develop-2019033116-7-g8604df4a
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v2.2.0-develop-2019033116-7-g8604df4a)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v2.2.0-develop-2019033116-7-g8604df4a/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v2.2.0-develop-2019033116-7-g8604df4a

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v2.2.0-develop-2019033116-7-ge2240f43

These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.

### job status
+ [live-roof-650-townsend runs](https://gnss-analysis.swiftnav.com/jobs/id=983133)
+ [live-roof-650-townsend-dropouts-zero-baseline runs](https://gnss-analysis.swiftnav.com/jobs/id=983134)

### gnss-analysis results
At least one run must complete for these links to have data.

#### passfail
+ [live-roof-650-townsend](https://gnss-analysis.swiftnav.com/metrics_preset=pass_fail&scenario=live-roof-650-townsend&summary_type=q50&groupby_key=firmware&firmware_versions=v2.2.0-develop-2019033116-7-ge2240f43&build_type=buildroot_pr&display_type=table)
+ [live-roof-650-townsend-dropouts-zero-baseline](https://gnss-analysis.swiftnav.com/metrics_preset=pass_fail&scenario=live-roof-650-townsend-dropouts-zero-baseline&summary_type=q50&groupby_key=firmware&firmware_versions=v2.2.0-develop-2019033116-7-ge2240f43&build_type=buildroot_pr&display_type=table)

#### detailed
+ [live-roof-650-townsend](https://gnss-analysis.swiftnav.com/metrics_preset=detailed&scenario=live-roof-650-townsend&summary_type=q50&groupby_key=firmware&firmware_versions=v2.2.0-develop-2019033116-7-ge2240f43&build_type=buildroot_pr&display_type=table)
+ [live-roof-650-townsend-dropouts-zero-baseline](https://gnss-analysis.swiftnav.com/metrics_preset=detailed&scenario=live-roof-650-townsend-dropouts-zero-baseline&summary_type=q50&groupby_key=firmware&firmware_versions=v2.2.0-develop-2019033116-7-ge2240f43&build_type=buildroot_pr&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v2.2.0-develop-2019033116-7-ge2240f43
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v2.2.0-develop-2019033116-7-ge2240f43)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v2.2.0-develop-2019033116-7-ge2240f43/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v2.2.0-develop-2019033116-7-ge2240f43

**Pull request merged**